source/content/terminus/02-install.md

```diff
@@ -133,71 +114,35 @@ You must log in with a machine token after the installation completes. A machine
 
 Commands that execute remote instructions to tools like Drush or WP-CLI require SSH authentication. Refer to [Generate and Add SSH Keys](/ssh-keys/) to prevent password requests when executing these commands.
 
-## Update Standalone Terminus
 
-You can update the [standalone Terminus PHAR](/terminus/install#standalone-terminus-phar) installation to the newest version with the command below.
+## Updating Terminus
 
-<Alert title="Warning" type="danger" >
+<Alert title="Note" type="info">
 
-The `self:update` command is only available for the standalone Terminus installation. Refer to the [command documentation](/terminus/commands/self-update) for available options.
+Terminus uses [Semantic versioning](https://semver.org/). Be sure to fully test compatibility with existing configurations before upgrading to new major releases.
 
 </Alert>
 
-```bash{promptUser: user}
-terminus self:update
-```
-
-## Update Terminus Installer PHAR
-
-You can update the Composer-managed version of Terminus that was installed with the [Terminus Installer PHAR](/terminus/install#terminus-installer-phar).
-
-1. Navigate to the directory where Terminus was originally installed.
-
-1. Run the following command:
-
-    ```bash{promptUser: user}
-    curl -O https://raw.githubusercontent.com/pantheon-systems/terminus-installer/master/builds/installer.phar && php installer.phar update
-    ```
-
-### Update Terminus Homebrew Installation
+### Homebrew Package Manager (Recommended for MacOS Users)
 
 You can update to the newest version of the [Homebrew installation](/terminus/install#homebrew-installation) by running the command below:
 
 ```bash{promptUser: user}
 brew upgrade pantheon-systems/external/terminus
 ```
 
-<Alert title="Note" type="info">
-
-Terminus uses [Semantic versioning](https://semver.org/). Be sure to fully test compatibility with existing configurations before upgrading to new major releases.
-
-</Alert>
-
-## Update Terminus with Plugin
-
-Use the [`self:plugin:update` command](/terminus/commands/self-plugin-update) if you use the Terminus plugin manager.
-
-## Troubleshooting
-
-### Nothing to install or update
-
-For Composer-managed Terminus installations, if the update command above returns an output that indicates no updates were found:
+### Update Standalone Terminus PHAR
 
-1. Delete the existing Terminus version (e.g. `$HOME/terminus`).
+You can update the [standalone Terminus PHAR](/terminus/install#standalone-terminus-phar) installation to the newest version with the command below.
 
-1. Re-run the install command:
+```bash{promptUser: user}
+terminus self:update
+```
 
-    ```bash{promptUser: user}
-    rm -rf $HOME/terminus
-    mkdir $HOME/terminus
-    cd $HOME/terminus
-    curl -O https://raw.githubusercontent.com/pantheon-systems/terminus-installer/master/builds/installer.phar
-    php installer.phar install
-    ```
 
-### Self:update not defined
+## Installing and Updating Terminus Plugins
```

Comment:
If we have a section for installing and updating terminus plugins, i would expect `terminus self:plugin:update` to be in the documentation, but in this case we just link out to the plugin directory.

Reply:
I say delete this section from this page, it's already documented [on another page](https://docs.pantheon.io/terminus/plugins). 

If the steps were different based on terminus installed via PHAR vs homebrew, then it would be helpful to cover plugin installation/updates in those contexts but otherwise it's just adding noise instead of value I think